token-metadata/program/src/utils.rs

```diff
@@ -692,17 +692,17 @@ pub fn spl_token_burn(params: TokenBurnParams<'_, '_>) -> ProgramResult {
 
 /// TokenBurnParams
 pub struct TokenBurnParams<'a: 'b, 'b> {
-    /// mint
+    /// CHECK: mint
```

Comment:
This might not actually be needed. I remember when trying to compile locally - I was getting a lot of errors due to these /// CHECK: keywords missing and I may have added it in a place that didn't need it.